src/Allegro.DotnetSdk/Allegro.DotnetSdk.csproj

```diff
@@ -1,13 +1,21 @@
-<?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.Build.NoTargets">
+
   <PropertyGroup>
     <TargetFramework>net6.0</TargetFramework>
```

Comment:
why still 6.0? you mentioned that this version support 8 only, not get it

Reply:
Sdk supports .NET SDK v8 only, but doesn't really care about TFM (so projects can still target `net6.0` but have to use SDK `8.0.100`+). It's because the CustomAfterBuildDirectoryProps hook was added only recently in the SDK.